src/main/java/io/cdap/plugin/gcp/publisher/source/BackoffConfig.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.gcp.publisher.source;
+
+import java.io.Serializable;
+
+/**
+ * Class used to configure exponential backoff for Pub/Sub API requests.
+ */
+public class BackoffConfig implements Serializable {
+  final int initialBackoffMs;
```

Comment:
these are better to be private, they probably were not before because it was an inner class.

Reply:
Done.